lib/AppInfo/composer.json

```diff
@@ -0,0 +1,14 @@
+{
+	"name": "vendor_name/AppInfo",
+	"description": "description",
+	"minimum-stability": "stable",
+	"license": "proprietary",
```

Comment:
Is this file needed?

Reply:
Ah thank you, no need, let me remove it!